app/services/users/profile.rb

```diff
@@ -204,6 +211,8 @@ def services
     def update_status_and_errors
       if scaffold.errors.present?
         scaffold.status = HTTP_SOME_ERRORS
+      elsif user.edipi.blank? || !user.loa3?
+        scaffold.errors = []
```

Comment:
Schema and Swagger tests failed initially but then passed with these adjustments to support new LOA3 + EDIPI checks.